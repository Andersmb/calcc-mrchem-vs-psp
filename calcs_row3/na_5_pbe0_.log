Starting job 5688401 on c1-26,c2-35,c10-[7,52],c11-60 at Fri May 20 11:39:27 CEST 2022

Copying na_5_pbe0_.out to /cluster/home/ambr/projects/psp/row3/calcs/doublets
Copying na_5_pbe0_.json to /cluster/home/ambr/projects/psp/row3/calcs/doublets

Task and CPU usage stats:
JobID           JobName  AllocCPUS   NTasks     MinCPU MinCPUTask     AveCPU    Elapsed ExitCode 
------------ ---------- ---------- -------- ---------- ---------- ---------- ---------- -------- 
5688401      na_5_pbe0_        180                                             00:18:39      0:0 
5688401.bat+      batch         24        1   00:00:03          0   00:00:03   00:18:39      0:0 
5688401.ext+     extern        180        5   00:00:00          3   00:00:00   00:18:39      0:0 
5688401.0      mrchem.x        180       30   00:18:14         29   00:41:55   00:18:22      0:0 

Memory usage stats:
JobID            MaxRSS MaxRSSTask     AveRSS MaxPages   MaxPagesTask   AvePages 
------------ ---------- ---------- ---------- -------- -------------- ---------- 
5688401                                                                          
5688401.bat+     19280K          0     19280K        0              0          0 
5688401.ext+          0          3          0        0              3          0 
5688401.0      6799156K         29 3511823564        0             26          0 

Disk usage stats:
JobID         MaxDiskRead MaxDiskReadTask    AveDiskRead MaxDiskWrite MaxDiskWriteTask   AveDiskWrite 
------------ ------------ --------------- -------------- ------------ ---------------- -------------- 
5688401                                                                                               
5688401.bat+       16.90M               0         16.90M        0.03M                0          0.03M 
5688401.ext+        0.00M               3          0.00M            0                3              0 
5688401.0           7.32M              21          7.12M       63.13M                0         37.88M 

Job 5688401 completed at Fri May 20 11:58:07 CEST 2022
