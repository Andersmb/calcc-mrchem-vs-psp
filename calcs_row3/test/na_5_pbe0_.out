                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                                                                     ***
***          __  __ ____   ____ _                                       ***
***         |  \/  |  _ \ / ___| |__   ___ _ __ ___                     ***
***         | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \                    ***
***         | |  | |  _ <| |___| | | |  __/ | | | | |                   ***
***         |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|                   ***
***                                                                     ***
***         VERSION            1.1.0-alpha                              ***
***                                                                     ***
***         Git branch         HEAD                                     ***
***         Git commit hash    e7d4b230bb6df8495b9a                     ***
***         Git commit author  Stig Rune Jensen                         ***
***         Git commit date    Sat Mar 19 14:14:51 2022 +0100           ***
***                                                                     ***
***         Contact: luca.frediani@uit.no                               ***
***                                                                     ***
***         Radovan Bast            Magnar Bjorgve                      ***
***         Roberto Di Remigio      Antoine Durdek                      ***
***         Luca Frediani           Gabriel Gerez                       ***
***         Stig Rune Jensen        Jonas Juselius                      ***
***         Rune Monstad            Peter Wind                          ***
***                                                                     ***
***************************************************************************

---------------------------------------------------------------------------

 MPI processes           :        (1 bank)                              30
 OpenMP threads          :                                               6
 Total cores             :                                             175
                                                                           
---------------------------------------------------------------------------

XCFun DFT library Copyright 2009-2020 Ulf Ekstrom and contributors.
See http://dftlibs.org/xcfun/ for more information.

This is free software; see the source code for copying conditions.
There is ABSOLUTELY NO WARRANTY; not even for MERCHANTABILITY or
FITNESS FOR A PARTICULAR PURPOSE. For details see the documentation.
Scientific users of this library should cite
U. Ekstrom, L. Visscher, R. Bast, A. J. Thorvaldsen and K. Ruud;
J.Chem.Theor.Comp. 2010, DOI: 10.1021/ct100117s

---------------------------------------------------------------------------

 MRCPP version         : 1.4.1
 Git branch            : HEAD
 Git commit hash       : 75d41879b1908a94a452
 Git commit author     : Stig Rune Jensen
 Git commit date       : Thu Jan 6 11:38:53 2022 +0100

 Linear algebra        : EIGEN v3.4.0
 Parallelization       : MPI/OpenMP (6 threads)

---------------------------------------------------------------------------


                                                                           
===========================================================================
                         MultiResolution Analysis
---------------------------------------------------------------------------
 polynomial order      : 9
 polynomial type       : Interpolating
---------------------------------------------------------------------------
 total boxes           : 8
 boxes                 : [          2           2           2 ]
 unit lengths          : [   32.00000    32.00000    32.00000 ]
 scaling factor        : [    1.00000     1.00000     1.00000 ]
 lower bounds          : [  -32.00000   -32.00000   -32.00000 ]
 upper bounds          : [   32.00000    32.00000    32.00000 ]
 total length          : [   64.00000    64.00000    64.00000 ]
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                        Initializing Molecule                        ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               2
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0      Na            :        0.000000        0.000000       -1.590853
    1       O            :        0.000000        0.000000        2.285920
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000        0.000000
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                 Computing Ground State Wavefunction                 ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial orbitals
 Method                  : Diagonalize SAD Hamiltonian
 Precision               : 1.00000e-04
 Screening               : 1.20000e+01 StdDev
 Restricted              : False
 Functional              : LDA (SVWN5)
 AO basis                : 3-21G
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                            Molecular Orbitals
---------------------------------------------------------------------------
 Alpha electrons         :                                              10
 Beta electrons          :                                               9
 Total electrons         :                                              19
---------------------------------------------------------------------------
    n  Occ Spin          :                                            Norm
---------------------------------------------------------------------------
    0    1    a          :                              9.999998955388e-01
    1    1    a          :                              9.999999775547e-01
    2    1    a          :                              1.000000004177e+00
    3    1    a          :                              9.999994948864e-01
    4    1    a          :                              9.999999882749e-01
    5    1    a          :                              9.999999893298e-01
    6    1    a          :                              1.000000357982e+00
    7    1    a          :                              1.000002073343e+00
    8    1    a          :                              1.000000093158e+00
    9    1    a          :                              1.000000098617e+00
   10    1    b          :                              9.999998878598e-01
   11    1    b          :                              9.999999768492e-01
   12    1    b          :                              1.000000004114e+00
   13    1    b          :                              9.999994952096e-01
   14    1    b          :                              9.999999882648e-01
   15    1    b          :                              9.999999893216e-01
   16    1    b          :                              1.000000353107e+00
   17    1    b          :                              1.000002079309e+00
   18    1    b          :                              1.000000093480e+00
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial energy
 Method                  : DFT (PBE0)
 Relativity              : None
 Precision               : 1.00000e-04
 Localization            : On
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                        Molecular Energy (initial)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                235.573919544085
 E-N energy              :            (au)               -613.654941472753
 Coulomb energy          :            (au)                142.728315741701
 Exchange energy         :            (au)                 -5.629453253505
 X-C energy              :            (au)                -17.446852272704
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                 22.699292156365
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)               -258.429011713176
 Nuclear energy          :            (au)                 22.699292156365
---------------------------------------------------------------------------
 Total energy            :            (au)             -2.357297195568e+02
                         :      (kcal/mol)             -1.479226311466e+05
                         :        (kJ/mol)             -6.189082891191e+05
                         :            (eV)             -6.414532166589e+03
===========================================================================


===========================================================================
                        Orbital Energies (initial)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)                -38.367915062430
    1    1    a          :            (au)                -18.547884762807
    2    1    a          :            (au)                 -2.269164747199
    3    1    a          :            (au)                 -1.178374026489
    4    1    a          :            (au)                 -1.177802719777
    5    1    a          :            (au)                 -1.171636181486
    6    1    a          :            (au)                 -0.581524283103
    7    1    a          :            (au)                 -0.044837794588
    8    1    a          :            (au)                  0.018413653815
    9    1    a          :            (au)                  0.035725385449
   10    1    b          :            (au)                -38.367865112493
   11    1    b          :            (au)                -18.525703714931
   12    1    b          :            (au)                 -2.269048013838
   13    1    b          :            (au)                 -1.177966292915
   14    1    b          :            (au)                 -1.177752225715
   15    1    b          :            (au)                 -1.171441480573
   16    1    b          :            (au)                 -0.501822235374
   17    1    b          :            (au)                  0.045220657467
   18    1    b          :            (au)                  0.062376423546
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -126.369002533444
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Optimize ground state orbitals
 Method                  : DFT (PBE0)
 Relativity              : None
 Checkpointing           : Off
 Max iterations          : 200
 KAIN solver             : 10
 Localization            : First two iterations
 Diagonalization         : Off
 Start precision         : 1.00000e-06
 Final precision         : 1.00000e-06
 Helmholtz precision     : Dynamic
 Energy threshold        : 1.00000e-06
 Orbital threshold       : 5.00000e-05
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
 Iter           MO residual             Total energy                Update
---------------------------------------------------------------------------
    0          4.358899e+00        -235.729719556811         -2.357297e+02
    1          5.900756e-01        -237.122267518867         -1.392548e+00
    2          4.105431e-01        -237.258930360101         -1.366628e-01
    3          3.175808e-01        -237.258253136195          6.772239e-04
    4          1.590341e-01        -237.301740766348         -4.348763e-02
    5          2.784523e-02        -237.303200030716         -1.459264e-03
    6          3.264725e-02        -237.303526914058         -3.268833e-04
    7          1.116555e-02        -237.303776273246         -2.493592e-04
    8          1.651964e-02        -237.303734029136          4.224411e-05
    9          8.024430e-03        -237.303837055445         -1.030263e-04
   10          6.728452e-03        -237.303856589795         -1.953435e-05
   11          5.164917e-03        -237.303862855751         -6.265956e-06
   12          1.598361e-03        -237.303865036844         -2.181092e-06
   13          1.490274e-03        -237.303866007152         -9.703079e-07
   14          6.119844e-04        -237.303866268503         -2.613512e-07
   15          4.645035e-04        -237.303866376297         -1.077939e-07
   16          1.616687e-04        -237.303866400788         -2.449167e-08
   17          1.002905e-04        -237.303866414934         -1.414543e-08
   18          5.345794e-05        -237.303866418259         -3.324885e-09
---------------------------------------------------------------------------
                      SCF converged in 18 iterations!
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                    Printing Molecular Properties                    ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               2
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0      Na            :        0.000000        0.000000       -1.590853
    1       O            :        0.000000        0.000000        2.285920
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000        0.000000
===========================================================================


===========================================================================
                         Molecular Energy (final)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                236.748196064840
 E-N energy              :            (au)               -614.871322288760
 Coulomb energy          :            (au)                140.977413242864
 Exchange energy         :            (au)                 -5.562971211734
 X-C energy              :            (au)                -17.294474381833
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                 22.699292156365
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)               -260.003158574624
 Nuclear energy          :            (au)                 22.699292156365
---------------------------------------------------------------------------
 Total energy            :            (au)             -2.373038664183e+02
                         :      (kcal/mol)             -1.489104232078e+05
                         :        (kJ/mol)             -6.230412111057e+05
                         :            (eV)             -6.457366882961e+03
===========================================================================


===========================================================================
                         Orbital Energies (final)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)                -38.559771524555
    1    1    a          :            (au)                -19.111252229275
    2    1    a          :            (au)                 -2.263889157435
    3    1    a          :            (au)                 -1.177730644077
    4    1    a          :            (au)                 -1.176901148754
    5    1    a          :            (au)                 -1.174289444670
    6    1    a          :            (au)                 -0.825811662548
    7    1    a          :            (au)                 -0.285775521304
    8    1    a          :            (au)                 -0.224727205387
    9    1    a          :            (au)                 -0.213412250658
   10    1    b          :            (au)                -38.559864169056
   11    1    b          :            (au)                -19.085253655613
   12    1    b          :            (au)                 -2.264107207632
   13    1    b          :            (au)                 -1.177647748967
   14    1    b          :            (au)                 -1.177181330040
   15    1    b          :            (au)                 -1.174222505000
   16    1    b          :            (au)                 -0.740057516890
   17    1    b          :            (au)                 -0.195906801342
   18    1    b          :            (au)                 -0.186700169963
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -129.574501893165
===========================================================================


===========================================================================
                           Dipole Moment (dip-1)
---------------------------------------------------------------------------
 r_O                     :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Electronic vector       :       -0.000004       -0.000007       -3.918554
 Magnitude               :            (au)                        3.918554
                         :         (Debye)                        9.959970
---------------------------------------------------------------------------
 Nuclear vector          :        0.000000        0.000000        0.787974
 Magnitude               :            (au)                        0.787974
                         :         (Debye)                        2.002829
---------------------------------------------------------------------------
 Total vector            :       -0.000004       -0.000007       -3.130581
 Magnitude               :            (au)                        3.130581
                         :         (Debye)                        7.957141
===========================================================================


                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                            Exiting MRChem                           ***
***                                                                     ***
***                       Wall time :  0h 18m 14s                       ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
